### PR TITLE
docs: sync k8s docs

### PR DESCRIPTION
**Pull request by ye11ow**
In the meanwhile, I removed the docs in gravity repo so that we don't maintain two copies. https://github.com/timeplus-io/gravity/pull/258/files

### CODE REVIEW

File: docs/k8s-helm.md
Comment (ye11ow) [inline docs/k8s-helm.md:39]: > Please choose the latest `CHART VERSION`

I think this "advice" is not necessary. Always use latest version should be the default mindset for most of the people. If people don't use latest version, they must have their own reasons (e.g. they want a stable build).

File: docs/k8s-helm.md
Comment (ye11ow) [inline docs/k8s-helm.md:50]: refresh this section based on the latest helm config

File: docs/k8s-helm.md
Comment (ye11ow) [inline docs/k8s-helm.md:80]: refresh this section based on the latest helm config

File: docs/k8s-helm.md
Comment (ye11ow) [inline docs/k8s-helm.md:119]: refresh this section based on the latest helm config

File: docs/k8s-helm.md
Comment (ye11ow) [inline docs/k8s-helm.md:140]: moved this section above

File: docs/k8s-helm.md
Comment (ye11ow) [inline docs/k8s-helm.md:150]: I don't think we need a dedicate section for user management as it is already part of UI feature. Instead we probably can have a section to mention how to deploy timeplus-cli pod and use cli. We can add this section once customer really uses timeplus-cli on k8s.

File: docs/k8s-helm.md
Comment (ye11ow) [inline docs/k8s-helm.md:78]: > Common values are

There is no such concept of `Common values` for storage class. Or put it into another word, whether it is a `Common values` or not does NOT impact the user's decision here. 

Besides, I believe`ebs-gp3-basic-auto-delete` and `ebs-gp3-basic-encrypted` are defined by ourselves. The user won't be able to find them even if they are using EKS

File: docs/k8s-helm.md
Comment (jovezhong) [inline docs/k8s-helm.md:10]: I use Zed to edit the docs repo and apply auto-format on save. So for example - will be changed to * for bullets and remove/add blank lines. Let's focus on the content review first. Before merging the PR, let me use Zed to save/format the file again to ensure the consistent markdown style.

File: docs/k8s-helm.md
Comment (jovezhong) [inline docs/k8s-helm.md:103]: let's keep it consistent. Line 86 uses $NS. Probably no need to change this line to ${NS}, because there is no special char, $NS is a bit cleaner.

File: docs/k8s-helm.md
Comment (jovezhong) [inline docs/k8s-helm.md:123]: Tech docs usually are clam and fact-based. Let's avoid "the powerful Timeplus Enterprise!"

File: docs/k8s-helm.md
Comment (jovezhong) [inline docs/k8s-helm.md:127]: dockerhub -> Docker Hub

Comment (jovezhong) [reply docs/k8s-helm.md:127]: In case ->In the case
user->users
kubernetes->Kubernetes
run->run:

File: docs/k8s-helm.md
Comment (jovezhong) [inline docs/k8s-helm.md:210]: I noticed you removed a few sections. Probably need add them back
- Uninstall and cleanup
- User management

File: docs/k8s-helm.md
Comment (jovezhong) [inline docs/k8s-helm.md:366]: For inter-site link, pls use /server_config..., without the hostname. This will help the doc tool to detect deadlink and handle i18n, but this will introduce a difference between markdown in docs and in helm chart.

**Review by jovezhong — CHANGES_REQUESTED** [review]
Leave a few comments, will take a second look after you resolve them.

File: docs/k8s-helm.md
Comment (jovezhong) [reply docs/k8s-helm.md:39]: Yes, good to remove 'choose the latest ..'

File: docs/k8s-helm.md
Comment (jovezhong) [reply docs/k8s-helm.md:150]: Previous version can define users in the helm chart, now it's part of the UI/CLI. I am not 100% sure we should remove this, since it's natural for the user to look up such user management when they are checking the deployment guide and this section is to provide the link to proper docs.

File: docs/k8s-helm.md
Comment (jovezhong) [reply docs/k8s-helm.md:78]: We dont need to mention `ebs-gp3-basic-auto-delete`, `ebs-gp3-basic-encrypted` But still need to highlight the user need to change this storage class. Maybe on AWS, also mention the recommended IOPS for EBS, say gp3, iot2 (is that 3000 as our recommendation?)

File: docs/k8s-helm.md
Comment (ye11ow) [reply docs/k8s-helm.md:10]: Can you share your formatter config? I can configure my format to follow the rules, otherwise we need to do this everytime

Comment (ye11ow) [reply docs/k8s-helm.md:10]: You can check if Zed support https://editorconfig.org/ or not.

Comment (jovezhong) [reply docs/k8s-helm.md:10]: https://zed.dev/docs/languages/markdown#format it only mentioned it will use Prettier to format the markdown. Since you will remove the markdown in other repo, this is the only copy. After we address other issues, I will format once and we don't need to reformat everything next time.

File: docs/k8s-helm.md
Comment (ye11ow) [reply docs/k8s-helm.md:210]: `Uninstall and cleanup` is moved above. For User management, let's use https://github.com/timeplus-io/docs/pull/264#discussion_r1898748254 to discuss

File: docs/k8s-helm.md
Comment (ye11ow) [reply docs/k8s-helm.md]: updated link

File: docs/k8s-helm.md
Comment (ye11ow) [reply docs/k8s-helm.md]: updated

File: docs/k8s-helm.md
Comment (ye11ow) [reply docs/k8s-helm.md]: updated

File: docs/k8s-helm.md
Comment (ye11ow) [reply docs/k8s-helm.md]: updated

File: docs/k8s-helm.md
Comment (ye11ow) [reply docs/k8s-helm.md:10]: Thanks. Then I'll leave it to you to do the final format before merging

File: docs/k8s-helm.md
Comment (ye11ow) [reply docs/k8s-helm.md:78]: I've added `Please check the [Planning capacity](#planning-capacity) section for storage recommendations` to this section.

File: docs/k8s-helm.md
Comment (ye11ow) [reply docs/k8s-helm.md:150]: >since it's natural for the user to look up such user management when they are checking the deployment guide and this section is to provide the link to proper docs

I don't know. I think this is very personal because at least I don't feel the same way. Generally speaking, I'd feel strange if I need to manage the user differently based on different deployment (think about Grafana). Or you can think this differently, say if a customer asking "How should I manage the user", we can simply reply with "you can go to user or use timeplus cli" without knowing how they deployed.
 
Anyway, I'll leave this to you. You can add this section if necessary.

**Pull request merged**
